packages/tools/installer/scripts/installer

```diff
@@ -634,20 +635,29 @@ do_poweroff() {
 }
 
 # setup needed variables
-INSTALLER_VERSION="0.2.8"
+INSTALLER_VERSION="0.3.0"
```

Comment:
Ok, let's do that. I think there's a lot of room for improvement even in this text based installer (change some text, ditch the repair/upgrade mode that does not even work on my system for some reason, etc), but maybe that takes a bit more discussion...